include/curl/multi.h

```diff
@@ -162,8 +162,9 @@ CURL_EXTERN CURLMcode curl_multi_fdset(CURLM *multi_handle,
 /*
  * Name:     curl_multi_wait()
  *
- * Desc:     Poll on all fds within a CURLM set as well as any
- *           additional fds passed to the function.
+ * Desc:     Poll on all fds within a CURLM set as well as any additional fds
+ *           passed to the function. If there is no file descriptor to wait
```

Comment:
> If there is no file descriptor to wait for

To me this means there are no fds in the multi-handle, but there are cases where fds are in the multi-handle, but we aren't waiting on them because we are rate limiting (CURLOPT_MAX_RECV_SPEED_LARGE) and the fds are in a conceptual paused/inactive state.

Maybe something like:

_If there are no valid or active file descriptors to wait on, this function will return immediately._

Reply:
Well, that's just a comment. I don't think it needs to be as detailed as the real man page of the function...

Reply:
That's fair, whichever is the most used method for documentation would make sense. For me it's the online documentation.

Updating the online docs could be a separate change since it's expanding on existing behavior